### DISCUSSION TITLE
Question

### DISCUSSION THREAD

**Discussion by Maroo1122** [SubtitleEdit/subtitleedit#8418] [General]
If I burned subtitles to blu-ray movie does it still blu-ray or it changes
I am confused about that

**Comment by niksedk** [comment]
I'm not sure what you mean... Blu-ray is a disc, like a dvd is a disc. 
Bllue-ray video can be  H.264.

To create a Blu-ray disc you might need a menu etc, so it would probably require some Blu-ray authoring software.

I'm no expert on this as I have not had an optical disc drive for many years.
